pytradfri/color.py

```diff
@@ -0,0 +1,48 @@
+from .const import (ATTR_LIGHT_COLOR_X as X, ATTR_LIGHT_COLOR_Y as Y)
+
+
+KNOWN_XY = {
+  2200: {X: 33135, Y: 27211},
+  2700: {X: 30140, Y: 26909},
+  4000: {X: 24930, Y: 24694},
+}
+
+KNOWN_KELVIN = KNOWN_XY.keys()
+
+KNOWN_X = [v[X] for v in KNOWN_XY.values()]
+
+
+def can_kelvin_to_xy(k):
+    return k is not None and min(KNOWN_KELVIN) <= k <= max(KNOWN_KELVIN)
```

Comment:
Let's remove the `None` check. We should check that before passing it to the util functions.